src/Resources/views/fields/content_fields.html.twig

```diff
@@ -0,0 +1,47 @@
+{% trans_default_domain "content_fields" %}
+
+{% extends "EzPublishCoreBundle::content_fields.html.twig" %}
+
+{% block ezobjectrelationlist_field %}
+    {% spaceless %}
+        {% if not ez_field_is_empty( content, field ) %}
+            <ul {{ block( 'field_attributes' ) }}>
+                {% for contentId in field.value.destinationContentIds if parameters.available[contentId] %}
+                     {{ fos_httpcache_tag('relation-' ~ contentId) }}
```

Comment:
Would be good if we could either somehow doctorate field output, or that we found a way to make this more native.
To avoid template overloading.